<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaptureTimeTest</class>
 <widget class="QWidget" name="CaptureTimeTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10,10,10,0">
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxCarrier">
     <property name="title">
      <string>载波双捕</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,2">
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>开始时间：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>结束时间：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>耗时       ：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTimeEdit" name="timeEditCarrierStartTime">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="displayFormat">
         <string>hh:mm:ss.zzz</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTimeEdit" name="timeEditCarrierStopTime">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="displayFormat">
         <string>hh:mm:ss.zzz</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="timeEditCarrierIntervalTime"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxDistance">
     <property name="title">
      <string>距离捕获</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,2">
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>开始时间：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>结束时间：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>耗时       ：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTimeEdit" name="timeEditDistanceStartTime">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="displayFormat">
         <string>hh:mm:ss.zzz</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTimeEdit" name="timeEditDistanceStopTime">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="displayFormat">
         <string>hh:mm:ss.zzz</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="timeEditDistanceIntervalTime"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxSystem">
     <property name="title">
      <string>系统捕获</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,2">
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>开始时间：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>结束时间：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>耗时       ：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTimeEdit" name="timeEditSystemStartTime">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="displayFormat">
         <string>hh:mm:ss.zzz</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTimeEdit" name="timeEditSystemStopTime">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="displayFormat">
         <string>hh:mm:ss.zzz</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="timeEditSystemIntervalTime"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxTestCtrl">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="title">
      <string>测试控制</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>154</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pBtnStart">
        <property name="text">
         <string>测试开始</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>154</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pBtnStop">
        <property name="text">
         <string>测试结束</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>154</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
